apps/etterlatte-behandling/src/main/kotlin/oppgave/OppgaveService.kt

```diff
@@ -48,16 +48,12 @@ class OppgaveService(
         oppgaveStatuser: List<String>,
         minOppgavelisteIdentFilter: String? = null,
     ): List<OppgaveIntern> =
-        if (bruker.saksbehandlerMedRoller.harRolleStrengtFortrolig()) {
```

Comment:
Burde denne sjekken gjøres uansett?

Reply:
Hvis vi åpner for bytting mellom adressebeskyttelse og ikke på enhter så må vi i hvertfall sjekke dette og slik det er nå i tilfelle sb bytter feil.